generated/nimodinst/attributes.py

```diff
@@ -5,15 +5,14 @@
 class Attribute(object):
     '''Base class for all typed attributes.'''
 
-    def __init__(self, attribute_id, default_channel=''):
+    def __init__(self, attribute_id):
         self._attribute_id = attribute_id
-        self._default_channel = default_channel
 
-    def __get__(self, obj, objtype):
-        return self.get(obj, self._default_channel)
+    def __get__(self, session, session_type):
+        return self.get(session, session._repeated_capability)
```

Comment:
This probably breaks niModInst.
We will know once the PR is posted and nimi-bot does its thing.